config/initializers/devise.rb

```diff
@@ -125,7 +125,9 @@
   # able to access the website for two days without confirming their account,
   # access will be blocked just in the third day. Default is 0.days, meaning
   # the user cannot access the website without confirming their account.
-  # config.allow_unconfirmed_access_for = 2.days
+
+  # a value bigger than 0 is required here for all our tests to work
```

Comment:
Why?

Reply:
because otherwise the accounts are not ready to use. You would have to manually click on the verification link / disable this verification for all the tests, for it to work

Reply:
Couldn't we add that as default to the factory with a trait `unconfirmed`

Reply:
would also be possible. On the other hand, (if i remember correctly) the PO did not dislike this setting in general. So we can leave it as it is anyways, and there is imho no need to adjust the tests so they would work without it